config/jobs/kubernetes-sigs/cluster-api/cluster-api-presubmits.yaml

```diff
@@ -1,5 +1,19 @@
 presubmits:
   kubernetes-sigs/cluster-api:
+  - name: pull-cluster-api-bazel-verify
+    always_run: true
+    labels:
+      preset-service-account: "true"
+    spec:
+      containers:
+      - image: gcr.io/k8s-testimages/cluster-api:v20180927-f09bc3b1b
+        args:
+        - "--repo=sigs.k8s.io/$(REPO_NAME)=$(PULL_REFS)"
+        - "--root=/go/src"
+        - "--upload=gs://kubernetes-jenkins/pr-logs"
+        - "--scenario=execute"
```

Comment:
Note: execute scenario jobs should eventually move to the pod-utils instead, it's less config!